scipion/__main__.py

```diff
@@ -74,7 +74,8 @@ def config2Dict(configFile, varDict):
         # For each section
         for sectionName, section in config.items():
             for variable, value in section.items():
-                varDict[variable] = expandvars(value)
+                # Expanding user and avoiding comments
+                varDict[variable] = expandvars(value).split('#')[0].strip()
```

Comment:
I think the expandvars is using value (that should still have the the #).
Probably is better:

varDict[variable] = expandvars(value.split('#')[0].strip())

??

Reply:
If you, @dmaluenda agree with my latest change I'll merge this.